staging/src/k8s.io/component-base/metrics/prometheus/controllers/metrics.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	k8smetrics "k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+var (
+	// Metrics provides access to all controllerreadiness metrics.
+	Metrics = newControllerMetrics()
```

Comment:
I would prefer exposing `NewControllerMetrics` instead of a global variable. If you are afraid that metrics might be registered twice, you can always add a global `sync.Once`, it is pretty common across the codebase.